### PR TITLE
Added Cubone + Marowak and two of their attacks, Bone Club + Bonemerang

### PR DESCRIPTION
**Pull request by dangerstone**
I put a little recoil on Bonemerang, because I thought that would be fun. :)

### CODE REVIEW

File: attacklist.py [hunk 1]
```diff
@@ -154,7 +154,8 @@
 fissure = Attack(name='fissure', attCategory='ground', baseDamage=300, pokemonLevel=0, baseCount=5, recoil=0, heal=0, accuracy=0.2)
 mudbomb = Attack(name='mud bomb', attCategory='ground', baseDamage=65, pokemonLevel=0, baseCount=10, recoil=0, heal=0, accuracy=0.85)
 mudshot = Attack(name='mud shot', attCategory='ground', baseDamage=55, pokemonLevel=0, baseCount=15, recoil=0, heal=0, accuracy=0.95)
-
+boneclub = Attack(name='bone club', attCategory='ground', baseDamage=65, pokemonLevel=0, baseCount=20, recoil=0, heal=0, accuracy=0.85)
+bonemerang = Attack(name='bonemerang', attCategory='ground', baseDamage=50, pokemonLevel=43, baseCount=10, recoil=10, heal=0, accuracy=0.9)
```
Comment (importhuman) [inline attacklist.py]: pokemonLevel has been set to 0 for all attacks, I don't remember what its purpose was, just figured I should point that out. :)

Comment (dangerstone) [reply attacklist.py]: Oh, thank you! I'll correct this right away. :)

File: attacklist.py [hunk 2]
```diff
@@ -154,7 +154,8 @@
 fissure = Attack(name='fissure', attCategory='ground', baseDamage=300, pokemonLevel=0, baseCount=5, recoil=0, heal=0, accuracy=0.2)
 mudbomb = Attack(name='mud bomb', attCategory='ground', baseDamage=65, pokemonLevel=0, baseCount=10, recoil=0, heal=0, accuracy=0.85)
 mudshot = Attack(name='mud shot', attCategory='ground', baseDamage=55, pokemonLevel=0, baseCount=15, recoil=0, heal=0, accuracy=0.95)
-
+boneclub = Attack(name='bone club', attCategory='ground', baseDamage=65, pokemonLevel=0, baseCount=20, recoil=0, heal=0, accuracy=0.85)
+bonemerang = Attack(name='bonemerang', attCategory='ground', baseDamage=50, pokemonLevel=0, baseCount=10, recoil=10, heal=0, accuracy=0.9)
```
Comment (Sahil-k1509) [inline attacklist.py:158]: xD there is a small typo in boomerang spelling. 
edit: my bad. The attack is called bonemerang -_-

**Review by Sahil-k1509 — APPROVED** [review]

File: attacklist.py (hunk 2 above)
Comment (dangerstone) [reply attacklist.py:158]: I feel you, man, I kept typing "boomerang" instead because "bonemerang" seriously looks like a typo xD

**Pull request merged**